apps/preview/next/pages/showcases/SelectDropdown/SelectDropdownPreselected.tsx

```diff
@@ -0,0 +1,113 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { useId, useRef, useState, type KeyboardEvent } from 'react';
+import classNames from 'classnames';
+import { SfIconExpandMore, SfListItem, SfDropdown, SfIconCheck, useTrapFocus } from '@storefront-ui/react';
+
+type OptionType = {
+  label: string;
+  value: string;
+};
+
+const options: OptionType[] = [
+  {
+    label: 'Startup',
+    value: 'startup',
+  },
+  {
+    label: 'Business',
+    value: 'business',
+  },
+  {
+    label: 'Enterprise',
+    value: 'enterprise',
+  },
+];
+
+export default function SelectDropdownWithPlaceholder() {
+  const [listVisible, setListVisible] = useState(false);
+  const [selectedOption, setSelectedOption] = useState<OptionType>(options[0]);
```

Comment:
I'm wondering if it's really worth to create a whole new example just to put this default value here in the `useState`. 🤔

Reply:
Let's leave it for now, some of the examples will be when we create form showcases page for required/error states.